microcosm_flask/tests/conventions/fixtures.py

```diff
@@ -81,6 +83,13 @@ def csv_column_order(self):
         return column_order
 
 
+@add_associated_schema(
+    "Foo",
+    [
+        "email",
+        "firstName",
+    ]
+)
 class PersonSchema(PersonCSVSchema):
```

Comment:
Example use of this new decorator. I added the `email` field in the `NewPerson` schema to have a non-required field.